src/Components/MainPage.jsx

```diff
@@ -1,22 +1,44 @@
-import React from 'react';
+import { Link } from 'react-router-dom';
+import { register } from 'swiper/element/bundle';
 import { useSelector } from 'react-redux';
+import React from 'react';
+import { BsTwitter } from 'react-icons/bs';
+import { BiLogoFacebook } from 'react-icons/bi';
+import { LiaGoogle } from 'react-icons/lia';
+import '../Style/Main.css';
+import 'swiper/css';
+
+register();
 
 function MainPage() {
-  const storageData = useSelector((state) => state.users);
+  const DataUser = useSelector((state) => state.users);
+  const DataExperiences = useSelector((state) => state.experiences);
 
   return (
     <div className="mainContainer">
-      <p>
-        Esto es la página principal para
-        {storageData.user.name}
-      </p>
+      <h1>{`Welcome ${DataUser.user.name}`}</h1>
+      <p className="secondtitle">at the best experiences in the world</p>
+      <swiper-container slides-per-view="3" navigation="true" class="swiper-container">
+        {DataExperiences.experiences.map((item) => (
+          <swiper-slide key={item.id}>
+            <Link className="cardlink" to={`./Register/${item.id}`}>
```

Comment:
@Dachrono I suggest changing the path to the Experience Details page. A possible path would be: `/experiences/${item.id}`

Reply:
Hello Carmen, tky for your time and your comments I appreciate it!